images/openshift-enterprise-cli.yml

```diff
@@ -1,16 +1,14 @@
 content:
   source:
     alias: ose
-    modifications:
-    - action: replace
-      match: origin-clients
-      replacement: atomic-openshift-clients
-    path: images/cli
+    dockerfile: images/cli/Dockerfile.rhel
```

Comment:
@smarterclayton - Please be aware: Doing it this way means it will copy the *entire* github repo contents into the distgit and then move `images/cli/Dockerfile.rhel` to `./Dockerfile` and will build with that in the root directory context. Is that what are you are intending here?

Reply:
Yes

Reply:
Or rather, let's put it this way.  Origin is using multi stage builds, and the multistage builds build different binaries within ose.

Reply:
Cool. Just had to be sure :)